### ISSUE TITLE
Readme vs class - DotSubGraph vs DotSubgraph

### ISSUE THREAD

**Issue by FelixReuthlinger** [vfrz/DotNetGraph#38]
This is just minor, but maybe worth correcting ;)

REAMDE says
```
var mySubGraph = new DotSubGraph().WithIdentifier("cluster_0");
```

Class is called 
```
DotSubgraph
```

**Comment by vfrz** [comment]
Nice catch, I'll fix this asap!

**Comment by vfrz** [comment]
Fixed by 8b5d8c6
